src/main/java/com/jarvisluong/chargingmap/controllers/HealthCheckController.java

```diff
@@ -0,0 +1,18 @@
+package com.jarvisluong.chargingmap.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class HealthCheckController {
```

Comment:
Nowaday you can use something like [actuator](https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/production-ready.html) instead of home made solution.